main/res/layout/preference_seekbar.xml

```diff
@@ -10,7 +10,7 @@
     tools:context=".settings.SettingsActivity" >
 
     <SeekBar
-        android:id="@+id/wp_threshold_seekbar"
```

Comment:
Was it intended to remove `wp_threshold_preference.xml` file?

Reply:
Yes, we already had two seekbar layout files (for wp_threshold and brouter_threshold), and this PR would have added a third one, although they are basically the same, so I refactored the two other settings to use the same layout file, giving it a generalized name.